include/WCSimDetectorConstruction.hh

```diff
@@ -71,6 +71,7 @@ public:
   G4String GetDetectorName()      {return WCDetectorName;}
   G4double GetWaterTubeLength()   {return WCLength;}
   G4double GetWaterTubePosition() {return WCPosition;}
+  G4double GetWaterTubeRadius()   {return WCRadius;}
```

Comment:
@federiconova Have you looked at [`WCSimDetectorConstruction::GetGeo_Dm(G4int i)`](https://github.com/WCSim/WCSim/blob/2f7ca11b89dc893f115932186184a94614e43ac2/src/WCSimConstructGeometryTables.cc#L350)? `GetGeo_Dm(3)` should give you the radius. (See e.g. [this example usage](https://github.com/WCSim/WCSim/blob/2f7ca11b89dc893f115932186184a94614e43ac2/src/WCSimRunAction.cc#L142).)